lib/views/home/screens/profile/profile_sections.dart

```diff
@@ -0,0 +1,311 @@
+import 'package:dw_flutter_app/components/tasks/standings_info_card.dart';
```

Comment:
w sumie to nie jestem fanem tych builderów, mogą zostać ale czemu po prostu nie zrobiłeś widgetów?

Reply:
Ja jestem właśnie fanem mniej kodu plus pokazuje mi to że one wypełniają część ui a nie są samodzielnym komponentem